### PR TITLE
Add version to init payload

### PR DESCRIPTION
**Pull request by rawbee**
Closes PP-4346

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pinwheel/react-native-pinwheel",
-  "version": "0.3.0",
+  "version": "0.4.0",
```
Comment (TrekSoft) [inline package.json:3]: You're going to update this to 2.0.0 right?

**Review by TrekSoft — APPROVED** [review]
cat-12-0

File: src/index.tsx [hunk 2]
```diff
@@ -82,6 +84,11 @@ export default ({linkToken, onSuccess, onExit, onEvent}: PinwheelProps) => {
             payload: { 
               platform: "${Platform.OS}",
               sdk: 'react native',
+              version: {
+                major: ${major},
+                minor: ${minor},
+                patch: ${patch}
+              },
```
Comment (roscioli) [inline src/index.tsx:91]: Can we add initializationOptions as well here?

**Comment by TrekSoft** [comment]
Added to my V2 PR - closing.

**Pull request closed without merging**